Home Assistant Core: migrate actions to dedicated pages (#46028)
home-assistant/home-assistant.github.io: source/_actions/homeassistant.restart.markdown

---
title: "Restart Home Assistant"
action: homeassistant.restart
domain: homeassistant
description: "Restarts Home Assistant after checking the configuration."
related_actions:
  - homeassistant.stop
  - homeassistant.check_config
  - homeassistant.reload_all
---

> Use this action to restart Home Assistant. Before it restarts, Home Assistant checks the configuration. If the check fails, the restart is cancelled so you are not left with a broken setup. A common use is to apply changes that only take effect after a full restart.

Reviewer: Use AmE spelling:

```suggestion
Use this action to restart Home Assistant. Before it restarts, Home Assistant checks the configuration. If the check fails, the restart is canceled so you are not left with a broken setup. A common use is to apply changes that only take effect after a full restart.
```

> You can also restart into safe mode, which starts Home Assistant without custom integrations and custom dashboard cards. This is useful when a custom add-on is preventing Home Assistant from starting normally.

Reviewer: Also, refactor "add-ons":

```suggestion
You can also restart into safe mode, which starts Home Assistant without custom integrations and custom dashboard cards. This is useful when one of these is preventing Home Assistant from starting normally.
```
